### ISSUE TITLE
App Mining latest payout schedule

### ISSUE THREAD

**Issue by muneeb-ali** [blockstack/app-mining#173]
Is the app mining payout schedule listed at https://app.co/mining the latest updated version?

> the total monthly payout in STX will begin in November at $500,000; and will increase to $700,000 in February 2020

**Comment by jeffdomke** [comment]
Good catch. Moving to App.co repo @GinaAbrams and @kyranjamie can you please update.
